### PR TITLE
J aimz/adah.james/feature/rider dashboard

### PR DESCRIPTION
**Pull request by J-aimz**

### CODE REVIEW

File: ryder/src/App.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
-import AppNavbar from "./components/landing_page/appNavbar";
-import LandingPage from "./components/landing_page/landingPage";
-import Footer from "./components/landing_page/footer";
+// import AppNavbar from "./components/AppNavbar";
```
Comment (PrinceJK) [inline ryder/src/App.js:3]: Always remove dead codes

File: ryder/src/App.js [hunk 2]
```diff
@@ -12,6 +13,12 @@ import "../src/styles/dashboard.css";
 // import UserNavbar from "./components/dashboard/userNavbar";
 // import RiderNavbar from "./components/dashboard/riderNavbar";
```
Comment (PrinceJK) [inline ryder/src/App.js:14]: Remove this too

File: ryder/src/App.js (hunk 1 above)
Comment (J-aimz) [reply ryder/src/App.js]: done sir

File: ryder/src/App.js (hunk 2 above)
Comment (J-aimz) [reply ryder/src/App.js]: done sir

**Review by PrinceJK — APPROVED** [review]

**Pull request merged**